help/mobile-notifications.md

```diff
@@ -167,6 +167,18 @@ To enable push notifications for your organization:
 
 {end_tabs}
 
+#### Warning banners
+
+To make sure that you are aware when your server does not have mobile
+notifications enabled, you will see a warning banner if:
```

Comment:
"you will see ..." => "the mobile apps will display a warning banner if:" -- we should be specific as to where this will appear.